cypress/integration/home.spec.js

```diff
@@ -0,0 +1,14 @@
+import {
+  buildDataCy,
+  LEARN_REACT_LINK_CYPRESS,
+} from '../../src/config/selectors';
+
+describe('Renders Home', () => {
+  it('should render the link', () => {
+    cy.visit('/');
+    cy.get(buildDataCy(LEARN_REACT_LINK_CYPRESS)).should(
+      'contain.text',
+      'Learn React',
+    );
+  });
+});
```

Comment:
Is it your test? In any case I think you're already planning to add tests, it can go in another PR if you want.